data/mods/gen9ssb/moves.ts

```diff
@@ -642,6 +642,31 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		type: "Flying",
 	},
 
+	// Frozoid
+	flatoutfalling: {
+		accuracy: 100,
+		basePower: 75,
+		category: "Physical",
+		shortDesc: "Typeless. Sets Gravity.",
+		name: "Flat out falling",
+		pp: 5,
+		priority: 0,
+		flags: {},
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Head Smash', target);
+			this.add('-anim', source, 'Gravity', target);
+		},
+		onHit(source) {
+			this.field.addPseudoWeather('gravity', source);
+		},
```

Comment:
```suggestion
		self: {
			onHit(source) {
				this.field.addPseudoWeather('gravity', source);
			},
		},
```

Reply:
not sure i follow here, since gravity is a field wide move, won't it be the same if it inflicts it on the user or the foe?

Reply:
onHit methods don't trigger sometimes when hit by a substitute. This just ensures that it'll always happen

Reply:
gotcha